addons/hr_recruitment_skills/models/hr_job.py

```diff
@@ -0,0 +1,30 @@
+from markupsafe import Markup
+from ast import literal_eval
+
+from odoo import fields, models, _
+
+
+class HrJob(models.Model):
+    _inherit = "hr.job"
+
+    skill_ids = fields.Many2many(comodel_name='hr.skill', string="Expected Skills")
+
+    def action_search_matching_applicant(self):
+        action = self.env['ir.actions.actions']._for_xml_id('hr_recruitment.crm_case_categ0_act_job')
+        context = literal_eval(action.get('context'))
+        context.update({'active_id': self.id})
+        action.update({
+            'name': _("Matching Applicants"),
+            'views': [
+                (self.env.ref('hr_recruitment_skills.crm_case_tree_view_inherit_hr_recruitment_skills').id, 'tree'),
+                (False, 'form'),
+            ],
+            'context': context,
+            'domain': [
+                ('job_id', '!=', self.id),
+                ('skill_ids', 'in', self.skill_ids.ids),
+            ],
+            'help': Markup('<p class="o_view_nocontent_empty_folder">No Matching Applicants</p>\
+                <p>We do not have any applicants who meet the skill requirements for this job position in the database at the moment.</p>')
```

Comment:
This has to be translated:
```suggestion
            'help': Markup(_('<p class="o_view_nocontent_empty_folder">No Matching Applicants</p>
                <p>We do not have any applicants who meet the skill requirements for this job position in the database at the moment.</p>')),
```

Reply:
Hello @Bertrand2 
Is it okay to use triple quotes ("""...""")?